zephyr/Kconfig

```diff
@@ -33,6 +33,16 @@ config MCUBOOT_BUILD_STRATEGY_FROM_SOURCE
 
 endchoice
 
+config MCUBOOT_VERSION
+        string "MCUbot version"
+        default "0.0.1"
+	help
+	  Value to be passed as 'version' argument to 'imgtool.py'.
+	  Note that no semantics are connected to this variable.
```

Comment:
> Note that no semantics are connected to this variable.

Is this likely to stay true? Should we say less here and reference external
documentation more?

Reply:
I am not aware of any documentation regarding this. Will take a second look.

Reply:
I can't find any docs about this other than the code. 
It will not stay like this for long in our fork at least, but this is something we can ensure to keep updated without much hassle.

```
➜  mcuboot git:(kconfig-version) ✗ rg -t rst "version"
ext/tinycrypt/documentation/tinycrypt.rst
140:    be changed in future versions of the library as there are applications
➜  mcuboot git:(kconfig-version) ✗ 
```